src/invest_reports/jinja_templates/sdr-ndr-report.html

```diff
@@ -12,28 +17,43 @@
 
   <h2 class="section-header">Results</h2>
 
-  {{ accordion_section(
-    'Results by Watershed',
-    wide_table(
+  {% if ws_vector_totals_table %}
```

Comment:
I think we should do `if ... is defined` because we are using strict templating rules that raise an error on references to undefined variables, unless checking `is defined`. https://jinja.palletsprojects.com/en/stable/api/#undefined-types

Although if it's always at least defined as `None` it probably does not raise the exception, but I haven't tested it.